tests/sflow_test.go

```diff
@@ -161,6 +160,47 @@ func TestSFlowAgent(t *testing.T) {
 
 		/* FIXME (nplanel) remove this Sleep when agent.FlushFlowTable() exist */
 		time.Sleep(2 * time.Second)
-		ts.Validate(t, &trace)
+		pcapTraceValidate(t, ts.GetFlows(), &trace)
+	}
+}
+
+func TestSFlowProbPath(t *testing.T) {
+	hostname, err := os.Hostname()
+	if err != nil {
+		t.Fatal(err.Error())
+	}
+
+	ts := NewTestStorage()
+
+	agent, analyzer := helper.StartAgentAndAnalyzerWithConfig(t, confAgentAnalyzer, ts)
+	defer agent.Stop()
+	defer analyzer.Stop()
+
+	time.Sleep(2 * time.Second)
+	setupCmds := []helper.Cmd{
+		{"ovs-vsctl add-br br-sflow", true},
+		{"ovs-vsctl add-port br-sflow sflow-intf1 -- set interface sflow-intf1 type=internal", true},
+		{"ip address add 169.254.33.33/24 dev sflow-intf1", true},
+		{"ip link set sflow-intf1 up", true},
+		{"ping -c 10 -I sflow-intf1 169.254.33.34", false},
```

Comment:
Well as I said before the purpose of this test is not to cover the layerpath which is cover by tests you wrote. This test is to cover the commit which fixes the issue about "probeGraph not filled". Just checking that the probeGraph is filled with the correct value whatever the layerpath value is should be enough.